deltachat-ffi/deltachat.h

```diff
@@ -318,7 +318,7 @@ char*           dc_get_blobdir               (const dc_context_t* context);
  *                    The library uses the `media_quality` setting to use different defaults
  *                    for recoding images sent with type DC_MSG_IMAGE.
  *                    If needed, recoding other file types is up to the UI.
- * - `basic_web_rtc_instance` = address to webrtc signaling server (https://github.com/cracker0dks/basicwebrtc)
+ * - `basic_webrtc_instance` = address to webrtc signaling server (https://github.com/cracker0dks/basicwebrtc)
```

Comment:
i would strike out the `basic` here - i think, this is the "string" that is added by the user to define the client to use. the "string" may come from provider-db, a qr-code-scan, may be entered manually or whatever.

i do not see that we will have multiple different configuration values here.
so, _if_ it is important to know the type of server before, i would just add these information to the string.

so, instead of 

```
basic_webrtc_instance = https://example.org/server/roomname=$ROOM
```

better

```
webrtc_instance = basicwebrtc:https://example.org/server/roomname=$ROOM
```

or so. when building the room-urls, core will remove that prefix. using accelerator or jitsi urls,  would not require this prefix.

Reply:
make sense to strike the "basic_" part here.